lib/scenic/adapters/postgres.rb

```diff
@@ -86,8 +99,10 @@ def drop_materialized_view(name)
       # @return [void]
       def refresh_materialized_view(name, concurrently: false)
         if concurrently
+          raise Scenic::Postgres::ConcurrentRefreshesNotSupportedError unless connection.supports_concurrent_refresh?
```

Comment:
We're only using this once, but I think this should be extracted to a helper like the other method just for consistency.